lib/src/rules/use_decorated_box.dart

```diff
@@ -0,0 +1,114 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'package:analyzer/dart/ast/ast.dart';
+import 'package:analyzer/dart/ast/visitor.dart';
+
+import '../analyzer.dart';
+import '../util/flutter_utils.dart';
+
+const _desc = r'Use `DecoratedBox`.';
+
+const _details = r'''
+
+**DO** use `DecoratedBox` when `Container` has only `Decoration`.
```

Comment:
Consider:

    **DO** use `DecoratedBox` when `Container` has only a `Decoration`.

Reply:
It's been changed, and I also added myself to `AUTHORS` 😏

Reply:
Thanks!